spec.html

```diff
@@ -13572,7 +13572,7 @@ <h1>Static Semantics: Early Errors</h1>
             a?.b
             `c`
           </code></pre>
-          <p>so that it would be interpreted as two valid statements. The purpose is to maintain consistency with similar code without optional chaining operator:</p>
+          <p>so that it would be interpreted as two valid statements. The purpose is to maintain consistency with similar code without optional chaining syntax:</p>
```

Comment:
```suggestion
          <p>so that it would be interpreted as two valid statements. The purpose is to maintain consistency with similar code without optional chaining:</p>
```

Reply:
Adopted; sorry that I should have made an issue first for this.